src/jackdaw/test/transports/kafka.clj

```diff
@@ -0,0 +1,201 @@
+(ns jackdaw.test.transports.kafka
+  (:require
+   [clojure.core.async :as async]
+   [clojure.tools.logging :as log]
+   [jackdaw.client :as kafka]
+   [jackdaw.data :as jd]
+   [jackdaw.test.commands :as cmd]
+   [jackdaw.test.transports :as t]
+   [jackdaw.test.serde :refer [apply-serializers apply-deserializers
+                               serde-map
+                               byte-array-serde]])
+  (:import
+   org.apache.kafka.streams.KafkaStreams$StateListener
+   org.apache.kafka.clients.consumer.ConsumerRecord
+   org.apache.kafka.clients.producer.ProducerRecord))
+
+(defn subscribe
+  "Subscribes to specified topics
+
+   `consumer` should be a kafka consumer
+   `topic-config` should be a sequence of topic-metadata maps"
+  [consumer topic-config]
+  (kafka/subscribe consumer topic-config))
```

Comment:
Is `subscribe` meant to be part of a public API? Why not just use `jackdaw.client/subscribe`?